plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/condition/ConsoleHasText.java

```diff
@@ -30,7 +31,11 @@ public ConsoleHasText() {
 	
 	@Override
 	public boolean test() {
-		return !consoleView.getConsoleText().isEmpty();
+		try {
```

Comment:
There is already ConsoleHasText condition in RedDeer... Wouldn't be better to make that condition good, instead of creating&maintaining your own condition (with the same behavior)?